### PR TITLE
Configuration 

### PR DESCRIPTION
**Pull request by chmette**
- setup Eslint with Airbnb's style guide and Prettier
- add server.js

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+  "tabWidth": 4,
+  "semi": false,
```
Comment (AdrianValdes) [inline .prettierrc.json]: Do we need all this in our prettierrc.json? Or with the `singleQuote` is enough? Could you check please, we have tabWidht in our prettier extension for VSCode.

File: README.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ An open source project to practice client-server intergration
  *  Mongoose.js
  *  Faker.js
 
-### test push 2
\ No newline at end of file
+### test push 3
```
Comment (AdrianValdes) [inline README.md]: We should remove this tests line, don't you think so? We don'T have to keep track manually there.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "records-server",
+  "version": "1.0.0",
+  "description": "An open source project to practice client-server intergration",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (AdrianValdes) [inline package.json]: We need a script for to run `eslint`, in the client we called `lint`, maybe we could call it the same here, what do you think?

Comment (AdrianValdes) [inline package.json]: It would be great to have a command to start the server, could you take care of it?

File: .prettierrc.json (hunk 1 above)
Comment (chmette) [reply .prettierrc.json]: Its not needed , this is basic config based, recommended on prettier.io . 
Check result:
.tabWidth in the extension (VSCode) is (in my personal case) set to 2 and semi to true. If your experience confirms a shorter config as sufficient lets do this, i was just working based on the documentation, cause i have no further experience to build on.

File: README.md (hunk 2 above)
Comment (chmette) [reply README.md]: removed. I was not aware of the one in the Backend project.

File: package.json (hunk 3 above)
Comment (chmette) [reply package.json]: done, implemented

Comment (chmette) [reply package.json]: done

File: .prettierrc.json (hunk 1 above)
Comment (AdrianValdes) [reply .prettierrc.json]: Yes, those things are made for people that don't have prettier at all, we don't need in our case I think. Now you can check what happens if you delete it and if it is all good leave it like that. We want to use as less things we don't need as possible. Thanks for checking

File: package.json (hunk 3 above)
Comment (AdrianValdes) [reply package.json]: Thanks for that.

Comment (AdrianValdes) [reply package.json]: Thanks, good job.

File: package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "description": "An open source project to practice client-server intergration",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint 'src/**/*.{js,jsx}' --config ./.eslintrc.json",
```
Comment (AdrianValdes) [inline package.json]: I think we are not going to have files with `jsx`  in the back-end, could you take care of remove them from here, we don't need to tell the linter we want to check that extension. Does that make sense?

**Review by AdrianValdes — COMMENTED** [review]
Good job, just one more comment.

File: .prettierrc.json (hunk 1 above)
Comment (chmette) [reply .prettierrc.json]: seem to work all fine

File: package.json (hunk 4 above)
Comment (AdrianValdes) [reply package.json]: Did we solve this issue?

Comment (chmette) [reply package.json]: done

**Review by AdrianValdes — APPROVED** [review]
Good work! Let's merge this in.

**Pull request merged**